src/test/shell/bazel/bazel_cc_code_coverage_test.sh

```diff
@@ -45,13 +45,28 @@ readonly COLLECT_CC_COVERAGE_SCRIPT=tools/test/collect_cc_coverage.sh
 
 # Setup to be run for every test.
 function set_up() {
+   mkdir -p "${COVERAGE_DIR_VAR}"
+
+  # COVERAGE_DIR has to be different than ROOT and PWD for the test to be
+  # accurate.
+  # Checking if the files have the same deviceID and inode is enough to
+  # determine if they are the same.
```

Comment:
Interesting, it may be useful to link the documentation explaining why this check is enough.

Reply:
Done.